resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1877 @@
+              value: small_intestine
+            - label: Spinal cord
+              value: spinal_cord
+            - label: Spleen
+              value: spleen
+            - label: Stomach
+              value: stomach
+            - label: Testis
+              value: testis
+            - label: Thymus
+              value: thymus
+            - label: Thyroid gland
+              value: thyroid_gland
+            - label: Trachea
+              value: trachea
+            - label: Tongue
+              value: tongue
+            - label: Uretar
+              value: uretar
+            - label: Urinary bladder
+              value: urinary_bladder
+            - label: Uterus
+              value: uterus
+            - label: Vagina
+              value: vagina
+            - label: Vasculature
+              value: vasculature
+            - label: Vein
+              value: vein
+            - label: Other
+              value: other
+            - label: N/A
+              value: na
+        - name: biosample_source
+          label: Biosample source
+          type: basic:string
+          description: |
+            General description of where your sample was derived from, e.g. osteosarcoma, vastus
+            lateralis muscle, etc.
+          required: false
+        - name: cell_line_source
+          label: Cell line source
+          description: |
+            Source of the cell line, e.g. Biobank, CCLE, patient derived, etc.
+          type: basic:string
+          required: false
+        - name: growth_protocol
+          label: Growth protocol
+          type: basic:string
+          description: |
+            Describe the conditions that were used to grow or maintain organisms or cells prior to
+            extract preparation.
+          required: false
+        - name: biosample_treatment
+          label: Biosample treatment
+          type: basic:string
+          description: |
+            If your sample was treated with a chemical agent search for the CHEBI number at
+            https://www.ebi.ac.uk/chebi/, e.g. cisplatin, CHEBI:27899.
+          required: false
+        - name: treatment_protocol
+          label: Treatment protocol
+          type: basic:string
+          description: |
+            Describe how your sample was treated, at different timepoints, if using non-chemical
+            treatments e.g. exercise, aging, etc.
+          required: false
+        - name: description
+          label: Description
+          type: basic:string
+          required: false
+        - name: quality_status
+          label: Quality status
+          type: basic:string
+          required: false
+          choices:
+            - label: Pass
+              value: PASS
+            - label: Warning
+              value: WARNING
+            - label: Fail
+              value: FAIL
+        - name: message
+          label: Message
+          type: basic:string
+          required: false
```

Comment:
These two fields were part of the nested group `qc` before. Prefix "quality" was added to "status". What about message?

Reply:
@OtonicarJan @uksurd88 Removal of the `qc` block (and renaming of the fields therein) is a regression in regards to the existing FE functionality. I vote for this to remain available in the `general` schema, as it allows for manual and possibly automated  marking of the sample QC status in the future.